Resources/Prototypes/Entities/Mobs/NPCs/space.yml

```
@@ -369,3 +369,201 @@
   suffix: "Salvage Ruleset"
   components:
     - type: SalvageMobRestrictions
+
+
+- type: entity
+  parent: SimpleSpaceMobBase
+  id: MobSnail
+  name: snail
+  description: Revolting unless you're french.
+  components:
+  - type: Body
+    prototype: Mouse
+  - type: GhostRole
+    makeSentient: true
+    allowSpeech: false
+    allowMovement: true
+    name: ghost-role-information-snail-name
+    description: ghost-role-information-snail-description
+    rules: ghost-role-information-freeagent-rules
+  - type: GhostTakeoverAvailable
+  - type: Sprite
+    drawdepth: SmallMobs
+    layers:
+    - map: ["enum.DamageStateVisualLayers.Base"]
+      state: snail
+      sprite: Mobs/Animals/snail.rsi
```

as i know you can move sprite path like this, and then remove from all children because they inherit the path
```suggestion
  - type: Sprite
    drawdepth: SmallMobs
    sprite: Mobs/Animals/snail.rsi
    layers:
    - map: ["enum.DamageStateVisualLayers.Base"]
      state: snail
```